src/core/sys/linux/sys/file.d

```diff
@@ -0,0 +1,35 @@
+/**
+ * D header file for Linux file ops.
+ *
+ * Copyright: Copyright Nemanja Boric 2016.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Nemanja Boric
+ */
+module core.sys.linux.sys.file;
+
+version (linux):
+extern (C):
+nothrow:
+@nogc:
+
+/* Operations for the `flock' call. */
+/// Shared lock
+enum LOCK_SH              = 0x01;
+/// Exclusive lock
+enum LOCK_EX              = 0x02;
+/// Unlock
+enum LOCK_UN              = 0x08;
+/// Atomic update
+enum __LOCK_ATOMIC        = 0x16;
```

Comment:
Hmm, I got it from mine (Ubuntu's) file.h, but I don't see it on glibc's upstream file.h, so I'll remove it.

Thanks and good catch!